### PR TITLE
Issue occurred when accessing the dashboard after session invalidation fixed

### PR DESCRIPTION
**Pull request by Buddhimah**
wso2/product-is#7011 public Fix

### CODE REVIEW

File: modules/features/org.wso2.identity.jaggery.apps.feature/src/main/resources/portal/gadgets/consent_management/js/gadget.js [hunk 1]
```diff
@@ -573,6 +573,50 @@ function constructDate(expiry) {
...
+                    } else {
+                        message({
+                            content: 'Error occurred while loading values for the grid.',
+                            type: 'error',
+                            cbk: function () {}
+                        });
+                    }
+                }
+            }
+
```
Comment (Abilashini) [inline modules/features/org.wso2.identity.jaggery.apps.feature/src/main/resources/portal/gadgets/consent_management/js/gadget.js]: Unnecessary blank line

File: modules/features/org.wso2.identity.jaggery.apps.feature/src/main/resources/portal/gadgets/user_auth_apps/js/gadget.js [hunk 2]
```diff
@@ -121,7 +121,51 @@ function itemRemoveValidate(appName) {
...
+                    } else {
+                        message({
+                            content: 'Error occurred while loading values for the grid.',
+                            type: 'error',
+                            cbk: function () {}
+                        });
+                    }
+                }
+            }
+
```
Comment (Abilashini) [inline modules/features/org.wso2.identity.jaggery.apps.feature/src/main/resources/portal/gadgets/user_auth_apps/js/gadget.js]: Unnecessary blank line

Comment (Buddhimah) [reply modules/features/org.wso2.identity.jaggery.apps.feature/src/main/resources/portal/gadgets/user_auth_apps/js/gadget.js]: FIXED : https://github.com/wso2/product-is/pull/7242

File: modules/features/org.wso2.identity.jaggery.apps.feature/src/main/resources/portal/gadgets/consent_management/js/gadget.js (hunk 1 above)
Comment (Buddhimah) [reply modules/features/org.wso2.identity.jaggery.apps.feature/src/main/resources/portal/gadgets/consent_management/js/gadget.js]: FIXED : https://github.com/wso2/product-is/pull/7242

File: modules/features/org.wso2.identity.jaggery.apps.feature/src/main/resources/portal/gadgets/user_auth_apps/js/gadget.js (hunk 2 above)
Comment (Buddhimah) [reply modules/features/org.wso2.identity.jaggery.apps.feature/src/main/resources/portal/gadgets/user_auth_apps/js/gadget.js]: FIXED : https://github.com/wso2/product-is/pull/7242

**Pull request merged**